spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  subject(:user) do
+    User.new(first_name: first_name, last_name: last_name, email: email, password: password, password_confirmation: password_confirmation)
+  end
+
+  let(:first_name)            { Faker::Name.first_name }
+  let(:last_name)             { Faker::Name.last_name }
+  let(:email)                 { Faker::Internet.email }
+  let(:password)              { 'password' }
+  let(:password_confirmation) { 'password' }
+
+  it do
+    is_expected.to be_valid
+  end
+
+  describe '#Generate users email' do
```

Comment:
`#create`

Reply:
Test by method of a model, if you are calling `#create` i assume there is a method name `created` in the model